en/resources/tutorials/templating.md

```diff
@@ -31,36 +29,83 @@ You can provide your own Kitura template engine by implementing the `TemplateEng
 
 ---
 
-## Adding a template engine to your Kitura app
+## Setting up
+See [Getting Started](/{{ page.lang }}/starter/gettingstarted.html) for instructions on how to install Kitura's command-line interface.
+
+To use the template engines we first need to setup a Kitura server. Open a terminal and run the following:  
+1. Navigate to Desktop  
+`cd ~/Desktop`
```

Comment:
Is there a reason we use Desktop? In food tracker we just do "cd ~"
(if you change this make sure it follows through the rest of the tutorial"

Reply:
This seemed to be the common place for tutorials to be placed, but if we use the home directory in FoodTracker then I think we should use the same here. I'll update, thanks!